cli/flox-rust-sdk/src/models/search.rs

```diff
@@ -115,6 +115,15 @@ impl std::fmt::Display for PathOrJson {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize, Default, PartialEq)]
+#[serde(rename_all = "kebab-case")]
+pub enum SearchStrategy {
+    Match,
+    MatchName,
```

Comment:
Not sure if we should just drop `Match` or `MatchName`, but adding `MatchNameOrRelPath` was pretty boilerplatey so I just went ahead and did it